code/client/src/sdk/mafia/framework/c_mafia_dbs.h

```diff
@@ -2,26 +2,18 @@
 
 #include "c_vehicles_database.h"
 
-#include "sdk/ue/c_weak_ptr.h"
-#include "sdk/ue/sys/sodb/c_sys_odb.h"
-
-#include <utils/hooking/hooking.h>
+#include "../../ue/c_weak_ptr.h"
+#include "../../ue/sys/sodb/c_sys_odb.h"
 
 namespace SDK {
     namespace mafia::framework {
         class C_MafiaDBs {
           public:
-            void GetTablesDatabase(ue::sys::sodb::C_SysODB *odb) {
-                const auto pattern = hook::get_opcode_address("E8 ? ? ? ? 48 8B F8 49 8B C7");
-                hook::this_call(pattern, this, odb);
-            }
+            void GetTablesDatabase(ue::sys::sodb::C_SysODB *);
 
             ue::C_WeakPtr<mafia::framework::C_VehiclesDatabase> GetVehiclesDatabase();
         };
 
-        static C_MafiaDBs *GetMafiaDBs() {
-            const auto pattern = hook::get_opcode_address("E8 ? ? ? ? 48 8D 55 87 48 8B C8");
-            return hook::call<C_MafiaDBs *>(pattern);
-        }
```

Comment:
I have a doubt on this. It was a static method. You're making a non static method of it, and also forgot passing the this pointer. Call will always fail.

@Greavesy1899 can you confirm that this setup works?

Reply:
I'm certain this should remain static. It's a getter function to fetch a global variable, so I'd keep it static.